pdns/dnsdistdist/meson/compiler-setup/meson.build

```
@@ -15,3 +15,8 @@ add_project_arguments(
 
 cxx = meson.get_compiler('cpp')
 system = target_machine.system()
+
+if cxx.get_id() == 'gcc' and cxx.version().version_compare('>=8.0,<9.0')
+  add_global_link_arguments('-lstdc++fs', language: ['c', 'cpp'])
```

```suggestion
  add_global_link_arguments('-lstdc++fs', language: ['cpp'])
```
I doubt this library can ever get used by pure C code.

---

Sigh. That must be why we can't have nice things.